include/convex_bodies/hpolytope.h

```diff
@@ -253,10 +253,11 @@ class HPolytope{
     int is_in(const Point &p) const {
         NT sum;
         int m = A.rows();
-        for (int i = 0; i < m; i++) {
-            sum = b(i);
-            for (unsigned int j = 0; j < _d; j++) sum -= A(i, j) * p[j];
+        const NT* b_data = b.data();
 
+        for (int i = 0; i < m; i++) {
+            sum = *b_data - A.row(i) * p.getCoefficients();
```

Comment:
Do we need variable sum here? Maybe check if *b_data - A.row(i) * p.getCoefficients() is negative and then increase b_data